tokio-trace/src/lib.rs

```diff
@@ -449,6 +449,23 @@ macro_rules! callsite {
 /// # }
 /// ```
 ///
+/// Creating a span with custom target and log level:
+/// ```
+/// # #[macro_use]
+/// # extern crate tokio_trace;
+/// # fn main() {
+/// span!(
+///     target: "app_span",
+///     level: tokio_trace::Level::TRACE,
+///     "my span",
+///     foo = 3,
+///     bar = "another string"
+/// ).enter(|| {
```

Comment:
TIOLI: We probably don't actually need to show entering the span in _all_ the examples, since the first one shows it, but I don't really care either way.